snapshot-labs/snapshot-sequencer
# fix: Use domain field

Related to https://github.com/snapshot-labs/workflow/issues/241

### Summery
- Use `domain` field instead of `domain` in settings JSON

### How to test:
- Change `domain` field and `settings.domain`
- Try to create proposal (it should verify new field instead of one in settings)
